pallets/loans/src/types.rs

```diff
@@ -47,13 +47,17 @@ pub enum MarketState {
 pub struct Market<Balance> {
     /// The collateral utilization ratio
     pub collateral_factor: Ratio,
+    /// A loose ratio than collateral_factor to avoid liquidate_borrow too casual
+    pub loose_collateral_factor: Ratio,
```

Comment:
Let's use liquidation_threshold, it's more clear.

Reply:
Yeah, I'm confused by this loose_collateral_factor.

Reply:
is the name `threshold` too vague? I think all current factors like `collateral_factor`, `close_factor` and be some kind of liquidation threshold